### DISCUSSION TITLE
Show all registered users

### DISCUSSION THREAD

**Discussion by infosion** [plausible/analytics#3372] [Self-Hosted Support] [answered]
Unfortunately I had the DISABLE_REGISTRATION option set to FALSE for a while and now I am concerned that users may have registered without my knowledge.

How can I see which users and which sites are present in Plausible? 

Unfortunately I have not been able to connect to the Postgres database. 

`docker-compose` exec plausible_db ash

`OCI runtime exec failed: exec failed: container_linux.go:344: starting container process caused "exec: \"ash\": executable file not found in $PATH": unknown`

Is there a solution or suggestion for this?
Thanks for the otherwise super good documentation.

**Comment by ruslandoga** [comment, marked as answer]
👋 @infosion 

Maybe `docker-compose exec plausible_db bash` would work? :)

**Reply by infosion** [reply]
Yes ;-) Many thanks!

**Comment by inigopascall** [comment]
Wait... this doesn't answer the question though? 😅

`docker-compose exec plausible_db bash` -- ok, nice, that's opened a bash prompt in the container... now what?

**Reply by ruslandoga** [reply]
👋 @inigopascall 

After you enter the container, you can use `psql` to run SELECTs against Plausible tables to get the information you need.

```console
$ cd hosting
$ docker compose exec plausible_db bash
```
```console
# psql -h localhost -U postgres
```
```sql
postgres=) \c plausible_db
-- You are now connected to database "plausible_db" as user "postgres".
plausible_db=) select count(*) from users;
--  count
-- -------
--      5
-- (1 row)
```

**Reply by inigopascall** [reply]
@ruslandoga thanks! it works :)
